src/utils/display-message.js

```diff
@@ -0,0 +1,11 @@
+const { EOL } = require( 'os' );
+
+const chalk = require( 'chalk' );
+const logSymbols = require( 'log-symbols' );
+
+module.exports = function displayMessage( type = 'info', message = 'Unexpected error' ) {
```

Comment:
I like this function, but let's call it `displayError` (and rename file to `display-error.js`) since it will be used only to display error messages.